test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala

```diff
@@ -242,6 +311,17 @@ class DeclarationTypeViewSpec extends UnitViewSpec with ExportsTestData with Dec
         val optionTwo = view.getElementById("Standard")
         optionTwo.attr("checked") mustBe empty
       }
+
+      "used for Simplified Declaration journey - Pre-lodged (F)" in {
+
+        val view = createView(formSimplified.fill(AdditionalDeclarationType.SIMPLIFIED_PRE_LODGED), DeclarationType.SIMPLIFIED)
+
+        val optionOne = view.getElementById("PreLodged")
+        optionOne.attr("checked") mustBe "checked"
```

Comment:
Helpers for checked and not checked we have in movements? It'll be good probably to move it to this service too.